source/release-notes.txt

```diff
@@ -10,8 +10,31 @@ Release Notes
    :depth: 1
    :class: twocols
 
-Relational Migrator 1.1.2
--------------------------
+1.1.3 Changelog
+---------------
+
+*Released July 26, 2023*
+
+New features:
+
+- Support for connecting to a SQL Server Named Instance.
+- Collection names now show for JSON schema files created from the :guilabel:`Code Generation` screen.
+
+Bug Fixes:
+
+- Fixes an issue where data verification after a sync job would sometimes 
+  stop or show as not started.
+- Fixes an issue where JSON columns could not be migrated if they contained 
+  top-level arrays.
```

Comment:
(minor tense tweaks)
Fixes an issue where JSON columns containing top-level arrays cannot be migrated.

Reply:
Updated.